src/components/Home-User-Meta.js

```diff
@@ -121,7 +121,7 @@ class HomeUserMeta extends Component {
           { this.userIsHirable() }
         </div>
         <div className="User-Meta-profile">
-          <img className="User-Meta-profile-picture" src={ this.userImg() } alt="user" />
+          <a href={ this.userUrl() }><img className="User-Meta-profile-picture" src={ this.userImg() } alt="user" /></a>
```

Comment:
Put this into a function that doesn't add url if a user hasn't been loaded yet.

e.g. `{this.getProfileImage()}`

Then just render with or without the link accordingly.